ui/src/app/pages/signed-in/signed-in.tsx

```diff
@@ -122,8 +121,6 @@ export const SignedInImpl = (spinnerProps: WithSpinnerOverlayProps) => {
     checkStoresLoaded();
   }, [profileState, tiers]);
 
-  const { enableDemographicSurveyV2Redirect } = environment;
```

Comment:
This was added to allow some environments (like Test) to ignore the forced-demo-survey feature, to enable tests to pass: #6939 / https://precisionmedicineinitiative.atlassian.net/browse/RW-8758 

I have taken the Demo Survey for all test users, and tests pass again.  Now all environments force users to take the demo survey, like Prod.